app/src/main/java/org/simple/clinic/bp/entry/LinearLayoutWithPreImeKeyEventListener.kt

```diff
@@ -7,13 +7,11 @@ import android.widget.LinearLayout
 
 class LinearLayoutWithPreImeKeyEventListener(context: Context, attrs: AttributeSet) : LinearLayout(context, attrs) {
 
-  var backKeyPressInterceptor: () -> Unit = {
-    throw AssertionError("Back-press interceptor not set")
-  }
+  var backKeyPressInterceptor: (() -> Unit)? = null
```

Comment:
Why was this made `nullable`? Didn't we add that assertion so that you are expected to override that property if you use this widget?

Reply:
I made this nullable as part of this change:

https://github.com/simpledotorg/simple-android/blob/5583e830ea9a475c586f7b7f31f960a69499d414/app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntrySheet.kt#L181

Reply:
Essentially clearing any reference from Rx to avoid leaks